tests/minifier.js

```diff
@@ -61,6 +61,35 @@
 
     // https://github.com/kangax/html-minifier/issues/229
     equal(minify('<CUSTOM-TAG></CUSTOM-TAG><div>Hello :)</div>'), '<custom-tag></custom-tag><div>Hello :)</div>');
+
+    input = '<tag v-ref:vm_pv :imgs=" objpicsurl_ "></tag>';
```

Comment:
We should link to an issue or describe what an issue is coz it's difficult to understand what's being tested otherwise.